cloud/k8s/deploy/k8s-deploy.sh

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright 2021 Kontain Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# main program for kontain-deploy DeamonSet
+
+die() {
+        msg="$*"
+        echo "ERROR: $msg" >&2
+        exit 1
+}
+
+function print_usage() {
+	echo "Usage: $0 [install/cleanup/reset]"
+}
+
+function install() {
+  # Copy kontain binaries to /opt/kontain
+  echo "copy kontain artifacts onto host"
+  [ -d /opt/kontain/bin ] || mkdir /opt/kontain/bin
+  cp -a /opt/kontain-artifacts/bin/km /opt/kontain/bin/km
+  cp -a /opt/kontain-artifacts/bin/krun /opt/kontain/bin/krun
```

Comment:
AFAIK there needs to be something extra here for selinux to allow krun/km to work

Reply:
For not I'll just add to TODO comment and we can come back to it. I've been reading a bit on Kubernetes Hosts/Workers/Nodes and I've see stuff all over the map on SELINUX from "disable it" to "you can use it if you know what you are doing."